index.html

```diff
@@ -690,6 +690,10 @@ <h3>Character</h3>
             but the above recommends that only one is used to define the characters.</div>
           </li>
         </ul>
+        <aside class="note">As indicated in [[[#foreign-elements-and-attributes]]], <code>ttm:agent</code>
+        elements can have foreign attributes and elements. This can be used to provide additional, proprietary
+        character information. This may be used for example to provide voice characteristics when
+        a particular speech engine is used.</aside>
```

Comment:
I don't think we should encourage proprietary metadata for voice selection when we have not resolved #121 given that SSML defines the [`<voice>`](https://www.w3.org/TR/speech-synthesis11/#S3.2.1) element and voice selection semantics.

Suggest a) deleting the last sentence and b) thinking about whether we actually want to specify a complex mapping to SSML voice selection based on attributes on the `<ttm:agent>` element specifically.